R/readWorkbook.R

```diff
@@ -69,43 +69,47 @@
 #' }
```

Comment:
why did you modify this file?

Reply:
Mostly from adding `assert_*()`s and while testing for any differences in changing up `write.xlsx()` as there were a lot of testing code that used the `write.xlsx(x, file); read.xlsx(file)` process